src/adapters/entity-adapter.ts

```diff
@@ -131,6 +131,19 @@ export class EntityAdapter implements IDdfAdapter {
     ));
   }
 
+  addIsClauseByKey(request) {
+    const isParentDomainPresent = entitySet => {
+      const domain = this.contentManager.domainHash[entitySet];
+
+      return includes(request.select.key, domain);
+    };
+    const extraIsClauses = request.select.key
+      .filter(concept => this.contentManager.conceptTypeHash[concept] === 'entity_set' && !isParentDomainPresent(concept))
+      .map(concept => ({$or: [{[`is--${concept}`]: 'TRUE'}, {[`is--${concept}`]: 'true'}]}));
```

Comment:
why are you working with string boolean values?

Reply:
I this case I use values as-is. Is it wrong?

Reply:
Does the reader do string comparison with the `is--` fields in the data? Or does it parse the queries `is--` fields to boolean?